.github/actions/matrix/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'Notify Matrix'
```

Comment:
I linked an existing action for this in the issue, can we not use that one?

Reply:
Actually, that action adds [a lot of unnecessary information to the message](https://github.com/KrazyKirby99999/matrix-commit/blob/cae66ad84edd6a0e5ce3e3a5e0532127d9a6a236/matrix-commit/action.py#L28-L33), so keep as is?

Reply:
Right, ok, I will probably work on a better version with the start of the OSTFL grant so this will do for now :+1: